hudi-utilities/pom.xml

```diff
@@ -450,6 +450,14 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- AWS Services -->
+    <!-- https://mvnrepository.com/artifact/software.amazon.awssdk/aws-java-sdk-sqs -->
+    <dependency>
+      <groupId>software.amazon.awssdk</groupId>
+      <artifactId>sqs</artifactId>
+      <version>${aws.sdk.version}</version>
+    </dependency>
```

Comment:
Same question as before: https://github.com/apache/hudi/pull/8441/files#r1164644285
Should this be moved to `hudi-aws` module (`hudi-utilities` module has already relied on `hudi-aws` module)?

Reply:
We have added these 2 dependencies back to the ```hudi-utililes``` pom because of the ```TestS3EventsSource```  ```java.lang.NoClassDefFoundError: org/apache/http/impl/client/DefaultClientConnectionReuseStrategy``` failure. We still need to identify that why these dependencies are required to be specified in ```hudi-utilities``` even though ```hudi-aws``` is pulling them in ```hudi-utililes```
```
<dependency>
      <groupId>org.apache.httpcomponents</groupId>
      <artifactId>httpclient</artifactId>
      <version>${aws.sdk.httpclient.version}</version>
    </dependency>
    <dependency>
      <groupId>org.apache.httpcomponents</groupId>
      <artifactId>httpcore</artifactId>
      <version>${aws.sdk.httpcore.version}</version>
    </dependency>
```